features/weed_reported_location.feature

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Nice feature file :)

Just a small thing, the formatting could do with a tidy up.